charts/kyverno/crds/crds.yaml

```diff
@@ -426,13 +427,14 @@ metadata:
 spec:
   additionalPrinterColumns:
   - JSONPath: .spec.background
-    description: Background provides choice for applying rules to existing resources.
+    description: Background controls if rules are applied to existing resources during
+      a background scan.
     name: Background
     type: string
   - JSONPath: .spec.validationFailureAction
-    description: ValidationFailureAction provides choice to enforce rules to resources
-      during policy application.
-    name: Validation FailureAction
+    description: ValidationFailureAction controls if a policy failure should not disallow
```

Comment:
`not disallow` should be `disallow` - may have typed it wrong earlier while editing...

Here is another attempt:

ValidationFailureAction controls if a policy failure should disallow (enforce) or allow and report (audit) the admission review request.

Reply:
Missed that either, updated.